charts/cromwell/templates/_deployment.tpl

```diff
@@ -152,4 +167,11 @@ spec:
           subPath: sqlproxy-service-account.json
           name: sqlproxy-ctmpls
           readOnly: true
+      initContainers:
+      - name: download-prometheusjmx-jar
+        image: alpine:3.12.0
+        command: ["wget", "-O", "/cromwell-prometheusjmx-jar/prometheusjmx.jar", "https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.13.0/jmx_prometheus_javaagent-0.13.0.jar"]
```

Comment:
Would like it if this jar could be baked into, like, a standard monitoring init container image that is used by multiple charts. But it definitely doesn't need to be done right away/before merging

Reply:
I agree, this would be the best approach long term. Will make a separate ticket to do this.